src/jit/lir.h

```diff
@@ -309,4 +309,19 @@ class LIR final
     static void InsertBeforeTerminator(BasicBlock* block, LIR::Range&& range);
 };
 
+inline void GenTree::SetIsUnusedValue()
+{
+    gtLIRFlags |= LIR::Flags::IsUnusedValue;
+}
+
+inline void GenTree::ClearIsUnusedValue()
+{
+    gtLIRFlags &= ~LIR::Flags::IsUnusedValue;
+}
+
+inline bool GenTree::IsUnusedValue() const
+{
+    return (gtLIRFlags & LIR::Flags::IsUnusedValue) != 0;
+}
+
```

Comment:
Naming: I wonder if the flag should be `LIR::Flags::UnusedValue` and the functions thereby `SetUnusedValue` / `ClearUnusedValue` / `IsUnusedValue`? Having the word `Is` on the flag itself seems linguistically wrong, and makes `SetIsUnusedValue` a tongue-twister.